libs/openFrameworks/3d/ofEasyCam.cpp

```diff
@@ -12,23 +12,41 @@ static const unsigned long doubleclickTime = 200;
 //----------------------------------------
 ofEasyCam::ofEasyCam(){
 	reset();
+	sensitivityTranslate = {1,1,1};
+	sensitivityRot = {1,1,1};
+}
+//----------------------------------------
+void ofEasyCam::enableOrtho(){
+	if (!getOrtho()) {
+		prevFarClip = getFarClip();
+		prevNearClip = getNearClip();
+	}
+	setNearClip(-10000);
```

Comment:
Right, it is super arbitrary. The reason I had was that the defaults were clipping the scene that I made for the example app. I'll remove this and maybe a good idea is to put it in the example. I didn't see the fact that it can be hard to debug, which could become very annoying.